lib/server.js

```diff
@@ -164,8 +172,7 @@ server.onPhantomPageCreate = function(req, res) {
     this.clearLocalStorage(req, res);
     this.clearMemoryCache(req, res);
 
-    req.prerender.page.run(_this.options.blockedResources || blockedResources, !!_this.options.logRequests, USER_AGENT, function(blockedResources, logRequests, ua, resolve, reject) {
-
+    req.prerender.page.run(args, function(options, resolve, reject) {
```

Comment:
@tuxnco I was refering to `this.settings.userAgent` in the anonymous function. Why the rest is not part of "settings" too